src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -193,9 +195,29 @@ public static boolean RequestHTTPS(
     if (res == SCHttpUtility.HTTPX_CODE_NO_ERROR) {
       success = true;
     }
+
+    appendTimeStampToFile(FileName);
     return success;
   }
 
+  /**
+   * Appends the current time to the end of a file for logging purposes.
+   *
+   * @param fileName the file path to append a time stamp to.
+   */
+  private static void appendTimeStampToFile(String fileName) {
+    long localTimeMillis = LocalTimeOffsetCalculator.getLocalTimeOffsetMilliseconds();
+    Date currentDate = new Date(localTimeMillis);

```

Comment:
What is the purpose of this section of code? I'm confused, because it looks like it is trying to append the current time (in millis) to the specified file, but it specifies `LocalTimeOffsetCalculator.getLocalTimeOffsetMilliseconds()` which is then passed to `new Date(long)` as `localTimeMillis`. Wouldn't that create a new Date object with the incorrect time? I would think the new Date() object would have some date around Jan 1970 since new Date() accepts the time since epoch and its only getting an offset.

Reply:
This was to help debugging by appending a changing value to the end of the file. I ended up dropping this commit for now.